TESTNET.md

```diff
@@ -0,0 +1,36 @@
+# How to set up a corda k8s node on the corda testnet network
+
+Azure steps :
```

Comment:
CHECKLIST.md already has cloud infrastructure setup steps, this part is not needed.

Reply:
done